include/asterisk/rtp_engine.h

```diff
@@ -311,6 +315,8 @@ struct ast_rtp_payload_type {
 	unsigned int primary_mapping:1;
 	/*! When the payload type became non-primary. */
 	struct timeval when_retired;
+	/*! Bitrate to over-ride mime type defaults */
+	int sample_rate;
```

Comment:
Looking at existing sample rate usage, they're unsigned int and we should be consistent with that across all usage so everywhere a sample rate exists, unsigned int